sql/core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.status.api.v1
+
+class ExecutionData (
+    val id : Long,
+    val status: String,
+    val description: String,
+    val planDescription: String,
+    val metrics: Seq[(String, String)],
```

Comment:
I'd prefer to have a small type representing a metric. A list of lists (which is what the JSON looks like) is a little too opaque.

Reply:
I have Seq of `Metric` now which contain name and value, is this ok.?